Source/WebCore/platform/graphics/ca/GraphicsLayerCA.cpp

```diff
@@ -340,6 +340,12 @@ Ref<PlatformCALayer> GraphicsLayerCA::createPlatformCALayer(PlatformLayer* platf
     return PlatformCALayerCocoa::create(platformLayer, owner);
 }
 
+Ref<PlatformCALayer> GraphicsLayerCA::createPlatformCALayerCustom(LayerHostingContextIdentifier, PlatformCALayerClient* owner)
+{
+    ASSERT_NOT_REACHED_WITH_MESSAGE("GraphicsLayerCARemote::createPlatformCALayerCustom should always be called instead of this, but this symbol is needed to compile WebKitLegacy.");
```

Comment:
This is unusual but maybe not bad?

Reply:
We do have a similar assert just a few lines down there - so I figured it'd be a good thing. At the very least it helps devs to see "okay this isn't where we'd normally come across".